From 68fa0df6d620afc79fd2d642a406b2c41cde9dc9 Mon Sep 17 00:00:00 2001
From: Antoine du Hamel <duhamelantoine1995@gmail.com>
Date: Tue, 20 Oct 2026 18:57:03 +0200
Subject: [PATCH] crypto: update root certificates to NSS 3.131.1
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This is the certdata.txt[0] from NSS 3.131.1.

Certificates removed:
- emSign ECC Root CA - G3
- emSign Root CA - C1
- emSign ECC Root CA - C3

[0] https://raw.githubusercontent.com/nss-dev/nss/refs/tags/NSS_3_131_1_RTM/lib/ckfw/builtins/certdata.txt
a749e413-a6b5-45ca-827d-a943355e84c8

PR-URL: https://github.com/nodejs/node/pull/66877
Refs: https://groups.google.com/a/mozilla.org/g/dev-tech-crypto/c/vSxZxnOMMvs
Reviewed-By: Richard Lau <richard.lau@ibm.com>
Reviewed-By: Ulises Gascón <ulisesgascongonzalez@gmail.com>
Reviewed-By: Ruy Adorno <ruy@vlt.sh>
Upstream-Commit: 68fa0df6d620afc79fd2d642a406b2c41cde9dc9
Backport-Note: Apply the targeted three-root removal to the existing
bundled store without replacing unrelated roots.
---
 src/node_root_certs.h |  51 ------
 tools/certdata.txt    | 366 ------------------------------------------
 2 files changed, 417 deletions(-)

diff --git a/src/node_root_certs.h b/src/node_root_certs.h
index 517dc8814c6d..01ed5af6ff60 100644
--- a/src/node_root_certs.h
+++ b/src/node_root_certs.h
@@ -1553,57 +1553,6 @@
 "fko=\n"
 "-----END CERTIFICATE-----",
 
-/* emSign ECC Root CA - G3 */
-"-----BEGIN CERTIFICATE-----\n"
-"MIICTjCCAdOgAwIBAgIKPPYHqWhwDtqLhDAKBggqhkjOPQQDAzBrMQswCQYDVQQGEwJJTjET\n"
-"MBEGA1UECxMKZW1TaWduIFBLSTElMCMGA1UEChMcZU11ZGhyYSBUZWNobm9sb2dpZXMgTGlt\n"
-"aXRlZDEgMB4GA1UEAxMXZW1TaWduIEVDQyBSb290IENBIC0gRzMwHhcNMTgwMjE4MTgzMDAw\n"
-"WhcNNDMwMjE4MTgzMDAwWjBrMQswCQYDVQQGEwJJTjETMBEGA1UECxMKZW1TaWduIFBLSTEl\n"
-"MCMGA1UEChMcZU11ZGhyYSBUZWNobm9sb2dpZXMgTGltaXRlZDEgMB4GA1UEAxMXZW1TaWdu\n"
-"IEVDQyBSb290IENBIC0gRzMwdjAQBgcqhkjOPQIBBgUrgQQAIgNiAAQjpQy4LRL1KPOxst3i\n"
-"AhKAnjlfSU2fySU0WXTsuwYc58Byr+iuL+FBVIcUqEqy6HyC5ltqtdyzdc6LBtCGI79G1Y4P\n"
-"PwT01xySfvalY8L1X44uT6EYGQIrMgqCZH0Wk9GjQjBAMB0GA1UdDgQWBBR8XQKEE9TMipuB\n"
-"zhccLikenEhjQjAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/BAUwAwEB/zAKBggqhkjOPQQD\n"
-"AwNpADBmAjEAvvNhzwIQHWSVB7gYboiFBS+DCBeQyh+KTOgNG3qxrdWBCUfvO6wIBHxcmbHt\n"
-"RwfSAjEAnbpV/KlK6O3t5nYBQnvI+GDZjVGLVTv7jHvrZQnD+JbNR6iC8hZVdyR+EhCVBCyj\n"
-"-----END CERTIFICATE-----",
-
-/* emSign Root CA - C1 */
-"-----BEGIN CERTIFICATE-----\n"
-"MIIDczCCAlugAwIBAgILAK7PALrEzzL4Q7IwDQYJKoZIhvcNAQELBQAwVjELMAkGA1UEBhMC\n"
-"VVMxEzARBgNVBAsTCmVtU2lnbiBQS0kxFDASBgNVBAoTC2VNdWRocmEgSW5jMRwwGgYDVQQD\n"
-"ExNlbVNpZ24gUm9vdCBDQSAtIEMxMB4XDTE4MDIxODE4MzAwMFoXDTQzMDIxODE4MzAwMFow\n"
-"VjELMAkGA1UEBhMCVVMxEzARBgNVBAsTCmVtU2lnbiBQS0kxFDASBgNVBAoTC2VNdWRocmEg\n"
-"SW5jMRwwGgYDVQQDExNlbVNpZ24gUm9vdCBDQSAtIEMxMIIBIjANBgkqhkiG9w0BAQEFAAOC\n"
-"AQ8AMIIBCgKCAQEAz+upufGZBczYKCFK83M0UYRWEPWgTywS4/oTmifQz/l5GnRfHXk5/Fv4\n"
-"cI7gklL35CX5VIPZHdPIWoU/Xse2B+4+wM6ar6xWQio5JXDWv7V7Nq2s9nPczdcdioOl+yuQ\n"
-"FTdrHCZH3DspVpNqs8FqOp099cGXOFgFixwR4+S0uF2FHYP+eF8LRWgYSKVGczQ7/g/IdrvH\n"
-"GPMF0Ybzhe3nudkyrVWIzqa2kbBPrH4VI5b2P/AgNBbeCsbEBEV5f6f9vtKppa+cxSMq9zwh\n"
-"bL2vj07FOrLzNBL834AaSaTUqZX3noleoomslMuoaJuvimUnzYnu3Yy1aylwQ6BpC+S5DwID\n"
-"AQABo0IwQDAdBgNVHQ4EFgQU/qHgcB4qAzlSWkK+XJGFehiqTbUwDgYDVR0PAQH/BAQDAgEG\n"
-"MA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBAMJKVvoVIXsoounlHfv4LcQ5\n"
-"lkFMOycsxGwYFYDGrK9HWS8mC+M2sO87/kOXSTKZEhVb3xEp/6tT+LvBeA+snFOvV71ojD1p\n"
-"M/CjoCNjO2RnIkSt1XHLVip4kqNPEjE2NuLe/gDEo2APJ62gsIq1NnpSob0n9CAnYuhNlCQT\n"
-"5AoE6TyrLshDCUrGYQTlSTR+08TI9Q/Aqum6VF7zYytPT1DU/rl7mYw9wC68AivTxEDkigcx\n"
-"HpvOJpkT+xHqmiIMERnHXhuBUDDIlhJu58tBf5E7oke3VIAb3ADMmpDqw8NQBmIMMMAVSKeo\n"
-"WXzhriKi4gp6D/piq1JM4fHfyr6DDUI=\n"
-"-----END CERTIFICATE-----",
-
-/* emSign ECC Root CA - C3 */
-"-----BEGIN CERTIFICATE-----\n"
-"MIICKzCCAbGgAwIBAgIKe3G2gla4EnycqDAKBggqhkjOPQQDAzBaMQswCQYDVQQGEwJVUzET\n"
-"MBEGA1UECxMKZW1TaWduIFBLSTEUMBIGA1UEChMLZU11ZGhyYSBJbmMxIDAeBgNVBAMTF2Vt\n"
-"U2lnbiBFQ0MgUm9vdCBDQSAtIEMzMB4XDTE4MDIxODE4MzAwMFoXDTQzMDIxODE4MzAwMFow\n"
-"WjELMAkGA1UEBhMCVVMxEzARBgNVBAsTCmVtU2lnbiBQS0kxFDASBgNVBAoTC2VNdWRocmEg\n"
-"SW5jMSAwHgYDVQQDExdlbVNpZ24gRUNDIFJvb3QgQ0EgLSBDMzB2MBAGByqGSM49AgEGBSuB\n"
-"BAAiA2IABP2lYa57JhAd6bciMK4G9IGzsUJxlTm801Ljr6/58pc1kjZGDoeVjbk5Wum739D+\n"
-"yAdBPLtVb4OjavtisIGJAnB9SMVK4+kiVCJNk7tCDK93nCOmfddhEc5lx/h//vXyqaNCMEAw\n"
-"HQYDVR0OBBYEFPtaSNCAIEDyqOkAB2kZd6fmw/TPMA4GA1UdDwEB/wQEAwIBBjAPBgNVHRMB\n"
-"Af8EBTADAQH/MAoGCCqGSM49BAMDA2gAMGUCMQC02C8Cif22TGK6Q04ThHK1rt0c3ta13FaP\n"
-"WEBaLd4gTCKDypOofu4SQMfWh0/434UCMBwUZOR8loMRnLDRWmFLpg9J0wD8ofzkpf9/rdcw\n"
-"0Md3f76BB1UwUCAU9Vc4CqgxUQ==\n"
-"-----END CERTIFICATE-----",
-
 /* Hongkong Post Root CA 3 */
 "-----BEGIN CERTIFICATE-----\n"
 "MIIFzzCCA7egAwIBAgIUCBZfikyl7ADJk0DfxMauI7gcWqQwDQYJKoZIhvcNAQELBQAwbzEL\n"
diff --git a/tools/certdata.txt b/tools/certdata.txt
index e5cc5b976d1f..30c721598589 100644
--- a/tools/certdata.txt
+++ b/tools/certdata.txt
@@ -12276,372 +12276,6 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
-#
-# Certificate "emSign ECC Root CA - G3"
-#
-# Issuer: CN=emSign ECC Root CA - G3,O=eMudhra Technologies Limited,OU=emSign PKI,C=IN
-# Serial Number:3c:f6:07:a9:68:70:0e:da:8b:84
-# Subject: CN=emSign ECC Root CA - G3,O=eMudhra Technologies Limited,OU=emSign PKI,C=IN
-# Not Valid Before: Sun Feb 18 18:30:00 2018
-# Not Valid After : Wed Feb 18 18:30:00 2043
-# Fingerprint (SHA-256): 86:A1:EC:BA:08:9C:4A:8D:3B:BE:27:34:C6:12:BA:34:1D:81:3E:04:3C:F9:E8:A8:62:CD:5C:57:A3:6B:BE:6B
-# Fingerprint (SHA1): 30:43:FA:4F:F2:57:DC:A0:C3:80:EE:2E:58:EA:78:B2:3F:E6:BB:C1
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "emSign ECC Root CA - G3"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\153\061\013\060\011\006\003\125\004\006\023\002\111\116\061
-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
-\040\120\113\111\061\045\060\043\006\003\125\004\012\023\034\145
-\115\165\144\150\162\141\040\124\145\143\150\156\157\154\157\147
-\151\145\163\040\114\151\155\151\164\145\144\061\040\060\036\006
-\003\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103
-\040\122\157\157\164\040\103\101\040\055\040\107\063
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\153\061\013\060\011\006\003\125\004\006\023\002\111\116\061
-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
-\040\120\113\111\061\045\060\043\006\003\125\004\012\023\034\145
-\115\165\144\150\162\141\040\124\145\143\150\156\157\154\157\147
-\151\145\163\040\114\151\155\151\164\145\144\061\040\060\036\006
-\003\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103
-\040\122\157\157\164\040\103\101\040\055\040\107\063
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\012\074\366\007\251\150\160\016\332\213\204
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\002\116\060\202\001\323\240\003\002\001\002\002\012\074
-\366\007\251\150\160\016\332\213\204\060\012\006\010\052\206\110
-\316\075\004\003\003\060\153\061\013\060\011\006\003\125\004\006
-\023\002\111\116\061\023\060\021\006\003\125\004\013\023\012\145
-\155\123\151\147\156\040\120\113\111\061\045\060\043\006\003\125
-\004\012\023\034\145\115\165\144\150\162\141\040\124\145\143\150
-\156\157\154\157\147\151\145\163\040\114\151\155\151\164\145\144
-\061\040\060\036\006\003\125\004\003\023\027\145\155\123\151\147
-\156\040\105\103\103\040\122\157\157\164\040\103\101\040\055\040
-\107\063\060\036\027\015\061\070\060\062\061\070\061\070\063\060
-\060\060\132\027\015\064\063\060\062\061\070\061\070\063\060\060
-\060\132\060\153\061\013\060\011\006\003\125\004\006\023\002\111
-\116\061\023\060\021\006\003\125\004\013\023\012\145\155\123\151
-\147\156\040\120\113\111\061\045\060\043\006\003\125\004\012\023
-\034\145\115\165\144\150\162\141\040\124\145\143\150\156\157\154
-\157\147\151\145\163\040\114\151\155\151\164\145\144\061\040\060
-\036\006\003\125\004\003\023\027\145\155\123\151\147\156\040\105
-\103\103\040\122\157\157\164\040\103\101\040\055\040\107\063\060
-\166\060\020\006\007\052\206\110\316\075\002\001\006\005\053\201
-\004\000\042\003\142\000\004\043\245\014\270\055\022\365\050\363
-\261\262\335\342\002\022\200\236\071\137\111\115\237\311\045\064
-\131\164\354\273\006\034\347\300\162\257\350\256\057\341\101\124
-\207\024\250\112\262\350\174\202\346\133\152\265\334\263\165\316
-\213\006\320\206\043\277\106\325\216\017\077\004\364\327\034\222
-\176\366\245\143\302\365\137\216\056\117\241\030\031\002\053\062
-\012\202\144\175\026\223\321\243\102\060\100\060\035\006\003\125
-\035\016\004\026\004\024\174\135\002\204\023\324\314\212\233\201
-\316\027\034\056\051\036\234\110\143\102\060\016\006\003\125\035
-\017\001\001\377\004\004\003\002\001\006\060\017\006\003\125\035
-\023\001\001\377\004\005\060\003\001\001\377\060\012\006\010\052
-\206\110\316\075\004\003\003\003\151\000\060\146\002\061\000\276
-\363\141\317\002\020\035\144\225\007\270\030\156\210\205\005\057
-\203\010\027\220\312\037\212\114\350\015\033\172\261\255\325\201
-\011\107\357\073\254\010\004\174\134\231\261\355\107\007\322\002
-\061\000\235\272\125\374\251\112\350\355\355\346\166\001\102\173
-\310\370\140\331\215\121\213\125\073\373\214\173\353\145\011\303
-\370\226\315\107\250\202\362\026\125\167\044\176\022\020\225\004
-\054\243
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "emSign ECC Root CA - G3"
-# Issuer: CN=emSign ECC Root CA - G3,O=eMudhra Technologies Limited,OU=emSign PKI,C=IN
-# Serial Number:3c:f6:07:a9:68:70:0e:da:8b:84
-# Subject: CN=emSign ECC Root CA - G3,O=eMudhra Technologies Limited,OU=emSign PKI,C=IN
-# Not Valid Before: Sun Feb 18 18:30:00 2018
-# Not Valid After : Wed Feb 18 18:30:00 2043
-# Fingerprint (SHA-256): 86:A1:EC:BA:08:9C:4A:8D:3B:BE:27:34:C6:12:BA:34:1D:81:3E:04:3C:F9:E8:A8:62:CD:5C:57:A3:6B:BE:6B
-# Fingerprint (SHA1): 30:43:FA:4F:F2:57:DC:A0:C3:80:EE:2E:58:EA:78:B2:3F:E6:BB:C1
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "emSign ECC Root CA - G3"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\060\103\372\117\362\127\334\240\303\200\356\056\130\352\170\262
-\077\346\273\301
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\316\013\162\321\237\210\216\320\120\003\350\343\270\213\147\100
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\153\061\013\060\011\006\003\125\004\006\023\002\111\116\061
-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
-\040\120\113\111\061\045\060\043\006\003\125\004\012\023\034\145
-\115\165\144\150\162\141\040\124\145\143\150\156\157\154\157\147
-\151\145\163\040\114\151\155\151\164\145\144\061\040\060\036\006
-\003\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103
-\040\122\157\157\164\040\103\101\040\055\040\107\063
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\012\074\366\007\251\150\160\016\332\213\204
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
-#
-# Certificate "emSign Root CA - C1"
-#
-# Issuer: CN=emSign Root CA - C1,O=eMudhra Inc,OU=emSign PKI,C=US
-# Serial Number:00:ae:cf:00:ba:c4:cf:32:f8:43:b2
-# Subject: CN=emSign Root CA - C1,O=eMudhra Inc,OU=emSign PKI,C=US
-# Not Valid Before: Sun Feb 18 18:30:00 2018
-# Not Valid After : Wed Feb 18 18:30:00 2043
-# Fingerprint (SHA-256): 12:56:09:AA:30:1D:A0:A2:49:B9:7A:82:39:CB:6A:34:21:6F:44:DC:AC:9F:39:54:B1:42:92:F2:E8:C8:60:8F
-# Fingerprint (SHA1): E7:2E:F1:DF:FC:B2:09:28:CF:5D:D4:D5:67:37:B1:51:CB:86:4F:01
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "emSign Root CA - C1"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\126\061\013\060\011\006\003\125\004\006\023\002\125\123\061
-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
-\040\120\113\111\061\024\060\022\006\003\125\004\012\023\013\145
-\115\165\144\150\162\141\040\111\156\143\061\034\060\032\006\003
-\125\004\003\023\023\145\155\123\151\147\156\040\122\157\157\164
-\040\103\101\040\055\040\103\061
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\126\061\013\060\011\006\003\125\004\006\023\002\125\123\061
-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
-\040\120\113\111\061\024\060\022\006\003\125\004\012\023\013\145
-\115\165\144\150\162\141\040\111\156\143\061\034\060\032\006\003
-\125\004\003\023\023\145\155\123\151\147\156\040\122\157\157\164
-\040\103\101\040\055\040\103\061
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\013\000\256\317\000\272\304\317\062\370\103\262
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\003\163\060\202\002\133\240\003\002\001\002\002\013\000
-\256\317\000\272\304\317\062\370\103\262\060\015\006\011\052\206
-\110\206\367\015\001\001\013\005\000\060\126\061\013\060\011\006
-\003\125\004\006\023\002\125\123\061\023\060\021\006\003\125\004
-\013\023\012\145\155\123\151\147\156\040\120\113\111\061\024\060
-\022\006\003\125\004\012\023\013\145\115\165\144\150\162\141\040
-\111\156\143\061\034\060\032\006\003\125\004\003\023\023\145\155
-\123\151\147\156\040\122\157\157\164\040\103\101\040\055\040\103
-\061\060\036\027\015\061\070\060\062\061\070\061\070\063\060\060
-\060\132\027\015\064\063\060\062\061\070\061\070\063\060\060\060
-\132\060\126\061\013\060\011\006\003\125\004\006\023\002\125\123
-\061\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147
-\156\040\120\113\111\061\024\060\022\006\003\125\004\012\023\013
-\145\115\165\144\150\162\141\040\111\156\143\061\034\060\032\006
-\003\125\004\003\023\023\145\155\123\151\147\156\040\122\157\157
-\164\040\103\101\040\055\040\103\061\060\202\001\042\060\015\006
-\011\052\206\110\206\367\015\001\001\001\005\000\003\202\001\017
-\000\060\202\001\012\002\202\001\001\000\317\353\251\271\361\231
-\005\314\330\050\041\112\363\163\064\121\204\126\020\365\240\117
-\054\022\343\372\023\232\047\320\317\371\171\032\164\137\035\171
-\071\374\133\370\160\216\340\222\122\367\344\045\371\124\203\331
-\035\323\310\132\205\077\136\307\266\007\356\076\300\316\232\257
-\254\126\102\052\071\045\160\326\277\265\173\066\255\254\366\163
-\334\315\327\035\212\203\245\373\053\220\025\067\153\034\046\107
-\334\073\051\126\223\152\263\301\152\072\235\075\365\301\227\070
-\130\005\213\034\021\343\344\264\270\135\205\035\203\376\170\137
-\013\105\150\030\110\245\106\163\064\073\376\017\310\166\273\307
-\030\363\005\321\206\363\205\355\347\271\331\062\255\125\210\316
-\246\266\221\260\117\254\176\025\043\226\366\077\360\040\064\026
-\336\012\306\304\004\105\171\177\247\375\276\322\251\245\257\234
-\305\043\052\367\074\041\154\275\257\217\116\305\072\262\363\064
-\022\374\337\200\032\111\244\324\251\225\367\236\211\136\242\211
-\254\224\313\250\150\233\257\212\145\047\315\211\356\335\214\265
-\153\051\160\103\240\151\013\344\271\017\002\003\001\000\001\243
-\102\060\100\060\035\006\003\125\035\016\004\026\004\024\376\241
-\340\160\036\052\003\071\122\132\102\276\134\221\205\172\030\252
-\115\265\060\016\006\003\125\035\017\001\001\377\004\004\003\002
-\001\006\060\017\006\003\125\035\023\001\001\377\004\005\060\003
-\001\001\377\060\015\006\011\052\206\110\206\367\015\001\001\013
-\005\000\003\202\001\001\000\302\112\126\372\025\041\173\050\242
-\351\345\035\373\370\055\304\071\226\101\114\073\047\054\304\154
-\030\025\200\306\254\257\107\131\057\046\013\343\066\260\357\073
-\376\103\227\111\062\231\022\025\133\337\021\051\377\253\123\370
-\273\301\170\017\254\234\123\257\127\275\150\214\075\151\063\360
-\243\240\043\143\073\144\147\042\104\255\325\161\313\126\052\170
-\222\243\117\022\061\066\066\342\336\376\000\304\243\140\017\047
-\255\240\260\212\265\066\172\122\241\275\047\364\040\047\142\350
-\115\224\044\023\344\012\004\351\074\253\056\310\103\011\112\306
-\141\004\345\111\064\176\323\304\310\365\017\300\252\351\272\124
-\136\363\143\053\117\117\120\324\376\271\173\231\214\075\300\056
-\274\002\053\323\304\100\344\212\007\061\036\233\316\046\231\023
-\373\021\352\232\042\014\021\031\307\136\033\201\120\060\310\226
-\022\156\347\313\101\177\221\073\242\107\267\124\200\033\334\000
-\314\232\220\352\303\303\120\006\142\014\060\300\025\110\247\250
-\131\174\341\256\042\242\342\012\172\017\372\142\253\122\114\341
-\361\337\312\276\203\015\102
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "emSign Root CA - C1"
-# Issuer: CN=emSign Root CA - C1,O=eMudhra Inc,OU=emSign PKI,C=US
-# Serial Number:00:ae:cf:00:ba:c4:cf:32:f8:43:b2
-# Subject: CN=emSign Root CA - C1,O=eMudhra Inc,OU=emSign PKI,C=US
-# Not Valid Before: Sun Feb 18 18:30:00 2018
-# Not Valid After : Wed Feb 18 18:30:00 2043
-# Fingerprint (SHA-256): 12:56:09:AA:30:1D:A0:A2:49:B9:7A:82:39:CB:6A:34:21:6F:44:DC:AC:9F:39:54:B1:42:92:F2:E8:C8:60:8F
-# Fingerprint (SHA1): E7:2E:F1:DF:FC:B2:09:28:CF:5D:D4:D5:67:37:B1:51:CB:86:4F:01
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "emSign Root CA - C1"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\347\056\361\337\374\262\011\050\317\135\324\325\147\067\261\121
-\313\206\117\001
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\330\343\135\001\041\372\170\132\260\337\272\322\356\052\137\150
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\126\061\013\060\011\006\003\125\004\006\023\002\125\123\061
-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
-\040\120\113\111\061\024\060\022\006\003\125\004\012\023\013\145
-\115\165\144\150\162\141\040\111\156\143\061\034\060\032\006\003
-\125\004\003\023\023\145\155\123\151\147\156\040\122\157\157\164
-\040\103\101\040\055\040\103\061
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\013\000\256\317\000\272\304\317\062\370\103\262
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
-#
-# Certificate "emSign ECC Root CA - C3"
-#
-# Issuer: CN=emSign ECC Root CA - C3,O=eMudhra Inc,OU=emSign PKI,C=US
-# Serial Number:7b:71:b6:82:56:b8:12:7c:9c:a8
-# Subject: CN=emSign ECC Root CA - C3,O=eMudhra Inc,OU=emSign PKI,C=US
-# Not Valid Before: Sun Feb 18 18:30:00 2018
-# Not Valid After : Wed Feb 18 18:30:00 2043
-# Fingerprint (SHA-256): BC:4D:80:9B:15:18:9D:78:DB:3E:1D:8C:F4:F9:72:6A:79:5D:A1:64:3C:A5:F1:35:8E:1D:DB:0E:DC:0D:7E:B3
-# Fingerprint (SHA1): B6:AF:43:C2:9B:81:53:7D:F6:EF:6B:C3:1F:1F:60:15:0C:EE:48:66
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "emSign ECC Root CA - C3"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\125\123\061
-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
-\040\120\113\111\061\024\060\022\006\003\125\004\012\023\013\145
-\115\165\144\150\162\141\040\111\156\143\061\040\060\036\006\003
-\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103\040
-\122\157\157\164\040\103\101\040\055\040\103\063
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\125\123\061
-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
-\040\120\113\111\061\024\060\022\006\003\125\004\012\023\013\145
-\115\165\144\150\162\141\040\111\156\143\061\040\060\036\006\003
-\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103\040
-\122\157\157\164\040\103\101\040\055\040\103\063
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\012\173\161\266\202\126\270\022\174\234\250
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\002\053\060\202\001\261\240\003\002\001\002\002\012\173
-\161\266\202\126\270\022\174\234\250\060\012\006\010\052\206\110
-\316\075\004\003\003\060\132\061\013\060\011\006\003\125\004\006
-\023\002\125\123\061\023\060\021\006\003\125\004\013\023\012\145
-\155\123\151\147\156\040\120\113\111\061\024\060\022\006\003\125
-\004\012\023\013\145\115\165\144\150\162\141\040\111\156\143\061
-\040\060\036\006\003\125\004\003\023\027\145\155\123\151\147\156
-\040\105\103\103\040\122\157\157\164\040\103\101\040\055\040\103
-\063\060\036\027\015\061\070\060\062\061\070\061\070\063\060\060
-\060\132\027\015\064\063\060\062\061\070\061\070\063\060\060\060
-\132\060\132\061\013\060\011\006\003\125\004\006\023\002\125\123
-\061\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147
-\156\040\120\113\111\061\024\060\022\006\003\125\004\012\023\013
-\145\115\165\144\150\162\141\040\111\156\143\061\040\060\036\006
-\003\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103
-\040\122\157\157\164\040\103\101\040\055\040\103\063\060\166\060
-\020\006\007\052\206\110\316\075\002\001\006\005\053\201\004\000
-\042\003\142\000\004\375\245\141\256\173\046\020\035\351\267\042
-\060\256\006\364\201\263\261\102\161\225\071\274\323\122\343\257
-\257\371\362\227\065\222\066\106\016\207\225\215\271\071\132\351
-\273\337\320\376\310\007\101\074\273\125\157\203\243\152\373\142
-\260\201\211\002\160\175\110\305\112\343\351\042\124\042\115\223
-\273\102\014\257\167\234\043\246\175\327\141\021\316\145\307\370
-\177\376\365\362\251\243\102\060\100\060\035\006\003\125\035\016
-\004\026\004\024\373\132\110\320\200\040\100\362\250\351\000\007
-\151\031\167\247\346\303\364\317\060\016\006\003\125\035\017\001
-\001\377\004\004\003\002\001\006\060\017\006\003\125\035\023\001
-\001\377\004\005\060\003\001\001\377\060\012\006\010\052\206\110
-\316\075\004\003\003\003\150\000\060\145\002\061\000\264\330\057
-\002\211\375\266\114\142\272\103\116\023\204\162\265\256\335\034
-\336\326\265\334\126\217\130\100\132\055\336\040\114\042\203\312
-\223\250\176\356\022\100\307\326\207\117\370\337\205\002\060\034
-\024\144\344\174\226\203\021\234\260\321\132\141\113\246\017\111
-\323\000\374\241\374\344\245\377\177\255\327\060\320\307\167\177
-\276\201\007\125\060\120\040\024\365\127\070\012\250\061\121
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "emSign ECC Root CA - C3"
-# Issuer: CN=emSign ECC Root CA - C3,O=eMudhra Inc,OU=emSign PKI,C=US
-# Serial Number:7b:71:b6:82:56:b8:12:7c:9c:a8
-# Subject: CN=emSign ECC Root CA - C3,O=eMudhra Inc,OU=emSign PKI,C=US
-# Not Valid Before: Sun Feb 18 18:30:00 2018
-# Not Valid After : Wed Feb 18 18:30:00 2043
-# Fingerprint (SHA-256): BC:4D:80:9B:15:18:9D:78:DB:3E:1D:8C:F4:F9:72:6A:79:5D:A1:64:3C:A5:F1:35:8E:1D:DB:0E:DC:0D:7E:B3
-# Fingerprint (SHA1): B6:AF:43:C2:9B:81:53:7D:F6:EF:6B:C3:1F:1F:60:15:0C:EE:48:66
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "emSign ECC Root CA - C3"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\266\257\103\302\233\201\123\175\366\357\153\303\037\037\140\025
-\014\356\110\146
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\076\123\263\243\201\356\327\020\370\323\260\035\027\222\365\325
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\125\123\061
-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
-\040\120\113\111\061\024\060\022\006\003\125\004\012\023\013\145
-\115\165\144\150\162\141\040\111\156\143\061\040\060\036\006\003
-\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103\040
-\122\157\157\164\040\103\101\040\055\040\103\063
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\012\173\161\266\202\126\270\022\174\234\250
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
 #
 # Certificate "Hongkong Post Root CA 3"
 #
